src/components/FolderList.jsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useState } from 'react';
+import { getFolderNameData } from '../apis/api';
+import AddIcon from '../assets/Add.svg';
+import Styles from '../styles/FolderPage.module.css';
+import SubHeader from './SubHeader';
+import AddFolderModal from './modals/AddFolder';
+
+const FolderList = () => {
+  const [name, setName] = useState([]);
```

Comment:
P1:
배열값의 state명이 단수인건 좀 이상하네요. 이름으로 값의 형식까지 알 수 있게 바꿔주시면 좋겠습니다.

```suggestion
  const [names, setNames] = useState([]);
```